image/descriptor.go

```diff
@@ -48,19 +50,36 @@ func (d *descriptor) hash() string {
 	return pts[1]
 }
 
+// Index implementation
+type Index v1.ImageIndex
```

Comment:
But why not just use `v1.ImageIndex`? You're not defining any methods so there should be no need to add this extra level of type indirection.